src/messages/IPreMessageSentEnrich.ts

```diff
@@ -0,0 +1,30 @@
+import { IHttp, IMessageExtend, IRead } from '../accessors';
+import { IMessage } from './IMessage';
+
+/**
+ * Handler called when a Rocketlet wants to enrich a message.
+ */
+export interface IPreMessageSentEnrich {
+    /**
+     * Enables the handler to signal to the Rocketlets framework whether
+     * this handler should actually be executed for the message
+     * about to be sent.
+     *
+     * @param message The message which is being sent
+     * @param read An accessor to the environment
+     * @param http An accessor to the outside world
+     * @return whether to run the execute or not
+     */
+    preMessageSentEnrichCheck?(message: IMessage, read: IRead, http: IHttp): boolean;
```

Comment:
`checkPreMessageSentEnirch` instead of what's currently there? I agree and was having issues reading it myself as it currently is